api/src/utils/csv-utils/csv-header-configs.ts

```diff
@@ -369,3 +368,122 @@ export const getLookupIdCellValidator = (
     ];
   };
 };
+
+/**
+ * Get the taxon header cell validator.
+ *
+ * Rules:
+ *  1. The cell must be a valid ITIS TSN or scientific name
+ *  2. The cell must be a valid species from the provided taxons
+ *
+ * @template StaticHeaderType
+ * @param {TaxonMap} taxonMap The list of taxons
+ * @param {CSVCellValidatorOptions} [options] cell validator options
+ * @returns {*} {CSVCellValidator} The validate cell callback
+ */
+export const getTaxonCellValidator = (taxonMap: TaxonMap, options?: CSVCellValidatorOptions): CSVCellValidator => {
```

Comment:
question: Can the `getTaxonRowValidator` call this function now?

Reply:
fixed